props/saber_BC_buttons.h

```diff
@@ -1567,6 +1589,25 @@ class SaberBCButtons : public PROP_INHERIT_PREFIX PropBase {
     if (scroll_presets_beep_delay_timer_.isTimerExpired()) {
         SaberBase::DoEffect(EFFECT_NEWFONT, 0);
     }
+    // Blade array changing helper. See id() for usage.
+    if (play_no_change_sound_) {
+      play_no_change_sound_ = false;
+
+      float real_id = PropBase::id(false);
+      size_t real_best_config = FindBestConfigForId(real_id);
```

Comment:
if no change happened, then this should be the same as current_config - blade.

Reply:
Yes, thanks.